vizro-core/examples/visual-vocabulary/pages/_factories.py

```diff
@@ -110,3 +110,33 @@ def column_and_line_factory(group: str):
             make_code_clipboard_from_py_file("column_and_line.py"),
         ],
     )
+
+
+def gantt_factory(group: str):
+    """Reusable function to create the page content for the gantt chart with a unique ID."""
+    return vm.Page(
```

Comment:
I guess you can remove this from the `_factories.py` (since it's not reused in multiple chart groups), and directly place it inside the `time.py`.